lib/hacluster.pm

```diff
@@ -326,13 +326,7 @@ sub check_cluster_state {
     assert_script_run "$crm_mon_cmd";
     assert_script_run "$crm_mon_cmd | grep -i 'no inactive resources'" if is_sle '12-sp3+';
     assert_script_run 'crm_mon -1 | grep \'partition with quorum\'';
-    if (is_sle('=12-sp2')) {
-        assert_script_run q/crm_mon -s | grep "$(crm node list | wc -l).*nodes online"/;
-    }
-    else {
-        assert_script_run q/crm_mon -s | grep "$(crm node list | grep -c ': member').*nodes online"/;
-        record_soft_failure 'bsc#1158180, regression in crm_mon -s output' if is_sle('>=15-sp2');
-    }
+    assert_script_run q/crm_mon -s | grep "$(crm node list | grep -c ': member') nodes online"/;
```

Comment:
You removed the 12-SP2 specific part, is the remaining line also works on this version?

Reply:
Agree. If it's not needed anymore, please add a 12-SP2 verification run just to be sure.

Reply:
You are right, TBH I wonder why I'd like to remove 12-SP2 part...